### PR TITLE
Add splash page proof of concept

### PR DESCRIPTION
**Pull request by kriskowal**
screen shot 2017-12-10 at 9 40 37 pm

### CODE REVIEW

File: proofs/splash/main.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	cur := display.Start
+	buf, cur = cur.Home(buf)
+	buf, cur = cur.Clear(buf)
+	buf, cur = cur.Hide(buf)
+
+Loop:
+	for {
+		splash(front)
+
+		buf, cur = display.Render(buf, cur, front, display.Model24)
```
Comment (jcorbin) [inline proofs/splash/main.go]: you're not doing differential rendering; just use `New` and drop the back buffer; same applies to the showroom proof fwiw

Comment (kriskowal) [reply proofs/splash/main.go]: Aye aye, can add it back when animated. Could be an opportunity to model car positions and movement choices with EPS.

**Comment by jcorbin** [comment]
image

**Comment by kriskowal** [comment]
Backèd out the back displays.

**Comment by kriskowal** [comment]
@jcorbin If you remove the lines that print the cars, does the line art at least realign? This may be related to #26, but more likely the terminal just has trouble with the glyphs.

**Comment by jcorbin** [comment]
Yes it does re-align in that case.

**Comment by kriskowal** [comment]
Added PoC for command channel.

**Comment by kriskowal** [comment]
Added clouds and animated.

**Comment by kriskowal** [comment]
Going to land. We can iterate.

**Pull request merged**